src/api/tracking/withdrawals.ts

```diff
@@ -44,7 +43,7 @@ export async function listNurseryWithdrawals(
     ],
     search: convertToSearchNodePayload(searchCriteria, organizationId),
     sortOrder: sortOrder ? [sortOrder] : [{ field: 'id', direction: 'Ascending' }],
-    count,
+    count: 1000,
```

Comment:
`hasNurseryWithdrawals` was the only place this had a value other than 1000, so I figured I'd simplify the function signature. Happy to undo this part of the change if we want to keep the parameter.